### PR TITLE
json file help message

### PR DESCRIPTION
**Pull request by pavertomato**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
 
 To see all available options, run: `wsi2dcm --help`.
 
+## Additional tags
```
Comment (dzelemba) [inline README.md]: I like your suggestion of just putting the output of wsi2dcm --help here, so users who are evaluating have a quick way to see the feature set.

Comment (pavertomato) [reply README.md]: i'll update this PR as https://github.com/GoogleCloudPlatform/wsi-to-dicom-converter/pull/33 will be merged to have complete set of params

File: README.md [hunk 2]
```diff
@@ -31,6 +31,45 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
 
 To see all available options, run: `wsi2dcm --help`.
 
+## Complite set of parametes
```
Comment (dzelemba) [inline README.md]: typo: Complete

File: README.md [hunk 3]
```diff
@@ -31,6 +31,45 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
...
+
+##### input
```
Comment (dzelemba) [inline README.md]: Can we improve these explanations in a similar fashion to the code comments we made in the last pull request? Also, let's use full sentences with proper punctuation.

File: README.md (hunk 2 above)
Comment (pavertomato) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (pavertomato) [reply README.md]: done, i think

File: README.md [hunk 4]
```diff
@@ -31,6 +31,52 @@ Test data is freely available from [OpenSlide](http://openslide.cs.cmu.edu/downl
...
+Input wsi file, supported by openslide.
+##### outFolder
+Folder to store dcm files
+##### tileHeight
+Tile height px.
+##### tileWidth
+Tile width px.
+##### levels 
+Number of levels to generate, levels == 0 means number of levels will be readed from wsi file.
```
Comment (dzelemba) [inline README.md]: typo: "will be read"

Comment (pavertomato) [reply README.md]: updated

**Pull request merged**